apps/component-date/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const summary = require('hof-behaviour-summary-page');
+
+module.exports = {
+  name: 'component-date',
+  baseUrl: '/component-date',
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
```

Comment:
It should not change in hof. 

`name` is *required* and needs to match the directory name. `baseUrl` is optional and should default to `/` since *most* apps will be a single app mounted at the domain root.